packages/bot/src/stores/index.js

```diff
@@ -1,11 +1,13 @@
 import FlyoutStore  from './flyout-store';
 import ScratchStore from './scratch-store';
+import ToolbarStore from './toolbar-store';
 
 export default class RootStore {
     constructor(core, ws) {
         this.core = core;
         this.ws = ws;
         this.flyout = new FlyoutStore();
+        this.toolbar = new ToolbarStore(this.flyout);
```

Comment:
@sam-binary you have access to `flyoutstore` from `rootstore` , dont need to pass it .